cli/ioctl/cmd/config/config_get_endpoint.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// configGetEndpointCmd represents the config get endpoint command
+var configGetEndpointCmd = &cobra.Command{
+	Use:       "get",
+	Short:     "Get endpoint for ioctl client",
+	ValidArgs: []string{"get", "endpoint"},
+	Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
can use ExactOneArg?

Reply:
yes, used ExactOneArg() in first version
this is second version, added ValidArgs: []string{"get", "endpoint"}, and calls OnlyValidArgs() to make sure the command can only contain "get endpoint", wrong args will report error